be/src/exec/pipeline/pipeline_driver.h

```diff
@@ -176,6 +191,10 @@ class PipelineDriver {
 private:
     // check whether fragment is cancelled. It is used before pull_chunk and push_chunk.
     bool _check_fragment_is_canceled(RuntimeState* runtime_state);
+    void _finishing_operator(OperatorPtr& op, RuntimeState* runtime_state);
+    void _finish_operator(OperatorPtr& op, RuntimeState* runtime_state);
```

Comment:
suggest to rename _finish_operator/_finishing_operator to _set_operator_finished/_set_operator_finishing.

finshing and finished  are participles and can be used as adjectives.

Reply:
ok